controllers/articles.controller.js

```diff
@@ -0,0 +1,13 @@
+const { fetchArticleByID } = require("../models/articles.model")
+
+exports.getArticleByID = (req, res, next) => {
+    const { article_id } = req.params;
+    fetchArticleByID(article_id)
+        .then((article) => {
+        if(!article){
+            return res.status(404).json({ msg: 'Not found'})
```

Comment:
This is not how to write a Promise.reject for a custom error message. Refer back to the error handling notes on how to do this.